src/App.tsx

```diff
@@ -15,71 +16,98 @@ export const goods = [
   'Garlic',
 ];
 
-export const App: React.FC = () => (
-  <main className="section container">
-    <h1 className="title">No goods selected</h1>
+interface State {
+  selectedGood: string,
+  noMessage: string,
```

Comment:
actually you don't need separate variable in state for the message that no selected goods. I meant you could write 'no goods selected' inside selectedGood state